UIAutomationTests/UnitTests.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.IO;
+using Xunit;
+
+namespace UIAutomationTests
+{
+    //https://xunit.net/docs/running-tests-in-parallel
+    public class UnitTests : IClassFixture<TestFixture>
+    {
+        TestFixture testFixture;
+        public UnitTests(TestFixture tf)
+        {
+            testFixture = tf;
+        }
+        [Fact]
+        public async void TestAW()
+        {
+            string MQueryExpression = File.ReadAllText("QueryPqFiles/ExasolAW.query.pq");
```

Comment:
seems all tests follow the same structure, and they are tightly couple to the production code structure.
Consider implementing a small API to decouple it.
For example https://github.com/exasol/elasticsearch-virtual-schema/blob/main/src/test/java/com/exasol/adapter/dialects/elasticsearch/ElasticSearchSqlDialectIT.java#L903

Reply:
Once again, good suggestion, but writing the automation tests has already taken up quite a bit of time. Maybe in the future if the need arises.